modules/time/ut-coverage/time_UT.c

```diff
@@ -820,16 +825,32 @@ void Test_Print(void)
     char               timeBuf[CFE_TIME_PRINTED_STRING_SIZE];
     char               expectedBuf[CFE_TIME_PRINTED_STRING_SIZE];
     CFE_TIME_SysTime_t time;
+    bool               usingDefaultEpoch = true;
 
     UtPrintf("Begin Test Print");
 
+    if (CFE_MISSION_TIME_EPOCH_YEAR != 1980 || CFE_MISSION_TIME_EPOCH_DAY != 1 || CFE_MISSION_TIME_EPOCH_HOUR != 0 ||
+        CFE_MISSION_TIME_EPOCH_MINUTE != 0 || CFE_MISSION_TIME_EPOCH_SECOND != 0)
+    {
+        UtPrintf("Custom epoch time requires manual inspection for CFE_TIME_Print");
+        usingDefaultEpoch = false;
+    }
+
     /* Test with zero time value */
     time.Subseconds = 0;
     time.Seconds    = 0;
 
     CFE_TIME_Print(timeBuf, time);
-    strcpy(expectedBuf, "1980-001-00:00:00.00000");
-    CFE_UtAssert_STRINGBUF_EQ(timeBuf, sizeof(timeBuf), expectedBuf, sizeof(expectedBuf));
+    if (usingDefaultEpoch)
+    {
+        strcpy(expectedBuf, "1980-001-00:00:00.00000");
+        CFE_UtAssert_STRINGBUF_EQ(timeBuf, sizeof(timeBuf), expectedBuf, sizeof(expectedBuf));
+    }
+    else
+    {
+        UtAssertEx(false, UTASSERT_CASETYPE_MIR, __FILE__, __LINE__, "%s",
+                   "MIR (Manual Inspection Required) for CFE_TIME_Print");
```

Comment:
For these instead of the constant string, could you replace with something like:
`Confirm adding seconds = %u, subseconds = %u to configured EPOCH results in time %s` or similar?  Then the manual inspection can just be on the results of the log.

Reply:
I liked your print.  I made the changes.